src/admin/config.yml

```diff
@@ -441,20 +441,80 @@ collections:
       - {label: "Project Listing Description", name: "description", widget: "markdown", required: false, editor_components: ["image"], hint: "This is text that appears for a project in the projects listing. This description is not required for sub-pages."}
       - {label: "External Project Website", name: "link", widget: "string", required: false, hint: "This is not required for sub-pages."}
       - {label: "Body", name: "body", required: false, widget: "markdown", hint: "This is the content that appears for a project page hosted on the IDRC site."}
-  - name: "tools"
-    label: "Tools"
-    label_singular: "Tool"
-    folder: "src/tools"
+  - name: "resources"
+    label: "Resources"
+    label_singular: "Resource"
+    folder: "src/resources"
     slug: "{{slug}}"
-    preview_path: "/tools/{{slug}}"
+    preview_path: "/resources/{{slug}}"
     create: true
     fields:
-      - {label: "Layout", name: "layout", widget: "hidden", default: "layouts/single--tool.njk"}
+      - {label: "Layout", name: "layout", widget: "hidden", default: "layouts/single--resource.njk"}
       - {label: "Title", name: "title", widget: "string", required: true}
-      - {label: "Short Name", name: "shortName", widget: "string", required: false}
       - {label: "Description", name: "description", widget: "markdown", required: true, editor_components: ["image"]}
-      - {label: "Tags", name: "tags", widget: "list", required: false}
-      - {label: "Link", name: "link", widget: "string", required: true}
+      - {label: "Published Year", name: "publishedYear", widget: "string", required: true}
```

Comment:
```suggestion
      - {label: "Published Year", name: "publishedYear", widget: "string", hint: "The year this resource was first publicly available.", required: true}
```

Reply:
Thanks @jhung, while you are at it, could you also provide hint text for updated date if there's going to be one?

Reply:
> Thanks @jhung, while you are at it, could you also provide hint text for updated date if there's going to be one?

I don't see a field for this information on the admin page, are we adding it to the UI? If we are adding this field, could it be a year field with a month selector / drop-down?

Ideally from the Admin, the user can enter the year and choose a month. When viewed it would look something like this:

![image](https://github.com/inclusive-design/idrc/assets/564762/1793f5b0-f279-42aa-bc4c-18f0e69cae2a)